pytripgui/tree_vc/TreeItems.py

```diff
@@ -62,12 +62,34 @@ def __init__(self, data=PatientModel(), parent=None, children=None):
         if parent and not isinstance(parent, PatientList):
             raise Exception("FieldItems can only be added to PlanItem")
 
+    def __repr__(self):
+        return self.data.name
+
+    def add_child(self, child):
+        if isinstance(child, PlanItem):
+            self.children += tuple([child])
```

Comment:
Why are you adding a tuple ?
BTW, single element tuple can be created with `(child,)`

Reply:
because of anytree self.children:

```
Traceback (most recent call last):
  File "/home/deerjelen/Dokumenty/IFJ/pytripgui/pytripgui/tree_vc/TreeController.py", line 24, in add_new_patient
    self._tree_model.add_patient(patient)
  File "/home/deerjelen/Dokumenty/IFJ/pytripgui/pytripgui/tree_vc/TreeModel.py", line 114, in add_patient
    self._root_item.add_child(patient)
  File "/home/deerjelen/Dokumenty/IFJ/pytripgui/pytripgui/tree_vc/TreeItems.py", line 65, in add_child
    self.children += ([child])
TypeError: can only concatenate tuple (not "list") to tuple
```
In baseclass 'children' field is a tuple, and you can concatenate it only with another tuple
https://github.com/c0fec0de/anytree/blob/d63289b65644c6e8c9c1d83d339c2c235d9ecc31/anytree/node/nodemixin.py#L244